### PR TITLE
Cobbduceus 1.1/X, Base(d) Healers (THIS KILLS TACTICAL KILLING AND CLONING???)

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
### Buffs C2 Healing
I should have leaned to more healing initially, especially since they're damaging. Goal here is to make them more enticing.

### Dilutes Starting Meds / Borg Hydro Microdosing

Perhaps this was the biggest issue in that your DEFAULTS carried way too much of the chem specifically designed for lower dosages so people were getting smacked with needless damage consistently.

You can still make your own "pure" samples I guess, but I think you'll like the current values.

Borgs can now switch between 5u/2u with altclick.

### Adds a C3 Diluent Specifically for Minor Brute/Burn

Wouldn't make much sense to dilute it with air! Granibitaluri is a new C3 that slowly tops off your damage by order of brute then burn. The catch is you must be at 10 damage or less for the healing to apply. ODs at 50.

This is the reagent all C2s are diluted with regardless of what they heal (meaning brute/burn chems will heal a tad better but only when you're about healed anyways).

## Why It's Good For The Game

Buffed C2s to make them more viable as contenders to some of the more oddity avenues you guys were using.

Dilutes C2s to make the roundstart items viable on their own without chemistry (or at least a chem master). Same with Borg Microdosing option.

Added the C3 since it was suggested by zxaber to have a form of healing more minor damage without going through the lengths of surgery.

Sorry in advance silicon players, the solutions are still pure in your hypo!

## Changelog
:cl: Anarcho-Capitalistabital-by
add: In General, Buffs healing thoroughput of C2s. You can stop paying your cloners overtime now!
tweak: The roundstart C2 methods are now diluted with a new reagent, Granibitaluri.
tweak: The borghypo now can switch to "microdosing", which is 2u transfer via AltClick
add: Granibitaluri is a C3 (no downside!!!) that tops off your brute THEN burn damage if they are at or  below 10 damage.
/:cl:

Thank you guys for being so patient and providing constructive feedback! G-g-g-good thing coders "play" haha!

### CODE REVIEW

**Comment by wesoda25** [comment]
i will keep saying it until i cant say it anymore, c2s should all damage the same organ (liver makes most sense). Also those healing values still aren't good enough, you should make it -3.

**Comment by ExcessiveUseOfCobblestone** [comment]
whoops forgot to push second commit

**Comment by Ty-the-Smonk** [comment]
KILL AND CLONE DEATH IS TEMPORARY KILL AND CLONE

**Comment by PKPenguin321** [comment]
I think people have been asking for a top-off chem ever since trekchem was removed so this is pretty welcome

**Comment by tralezab** [comment]
nobody has been asking for that name though, that is one terrible chem name

**Comment by anconfuzedrock** [comment]
one hit with a toolbox and it won't heal so this changes nothing

**Review by tralezab — COMMENTED** [review]
just a reminder to fix some pathing

**Comment by dr-bee** [comment]
Id rather just eat something. Vitamin is significantly better as a healer and easier to find. This isnt really a solution.

**Comment by anconfuzedrock** [comment]
when do we make sleeping heal people

**Comment by ExcessiveUseOfCobblestone** [comment]
> 
> 
> Id rather just eat something. Vitamin is significantly better as a healer and easier to find. This isnt really a solution.

 food healing for 1/1 brute and burn 50% for no reason just goes to show how flippantly people just tossed healing around.

Just having 10u vitamin gives you 25 iterations which roughly translates to about 12.5 HP of both brute AND burn. stupid tbh

**Comment by Krysonism** [comment]
The name of the new chem is horrible. That is all, thanks.

**Comment by cacogen** [comment]
i don undesa

**Comment by PKPenguin321** [comment]
Cobbducobbyeus 1.1/C, Cobb(y) Cobbers (THIS COBBS COBTICAL COBBING AND CLOBBING?C?)

**Comment by ExcessiveUseOfCobblestone** [comment]
did I mention cobby was the author or this PR?

**Comment by Dawson1917** [comment]
>The catch is you must be at 10 damage or less for the healing to apply. 

just add passive healing at this point. you don't need chems for accidentally touching a lightbulb.

**Comment by ExcessiveUseOfCobblestone** [comment]
I made the healing after I added the diluent since I didn't want things to just look like 2u of a chem.

**Comment by AdamElTablawy** [comment]
what use is a chemical specifically for healing pathetic amounts of damage

at that point eat food or drink milk or something unless you've got an issue with milk healing people now

**Comment by FancyCyclops** [comment]
This doesn't nerf the usage of these chems in murder, one of the most obvious ways to use these.

**Comment by Difrit** [comment]
Can we at least stop making the new chem names sound like someone having a stroke

**Comment by Akrilla** [comment]
medical shapin up

**Comment by nemvar** [comment]
Granibitaluri seems a tad weak. As in... it might as well do nothing. 
If you take 50 units of it, it heals a whole of 12.5 damage. Compare that to lets say... nutriment, which heals half a brute damage per tick on average and as such is 5 times as strong as granibitaluri.
Nobody in their right mind will EVER use a medicine with a healing amount that is that low. If it only heals tiny boo boos, it may aswell have a decent heal rate of one (WHOLE) damage per second otherwise it is literally outclassed by every single food reagent in the game.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -1234,3 +1234,24 @@
...
+	name = "Granibitaluri" //achieve "GRANular" amounts of C2
+	description = "A chemical solution useful as a diluent for more potent medicine. Especially useful when combined with brute/burn medication. Large amounts can cause fluid to appear in both the lungs and the heart."
+	reagent_state = LIQUID
+	overdose_threshold = 50
+	metabolization_rate = 0.2 //same as C2s
+
+/datum/reagent/medicine/granibitaluri/on_mob_life(mob/living/carbon/M) //
+	if(M.getBruteLoss() <= 10)
+		M.adjustBruteLoss(-0.1)
+	else if(M.getFireLoss() <= 10)
```
Comment (nemvar) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: This won't heal fireloss if bruteloss is 0. Is that intentional?

File: code/modules/reagents/reagent_containers/borghydro.dm [hunk 2]
```diff
@@ -152,6 +153,15 @@ Borg Hypospray
 	if(empty)
 		. += "<span class='warning'>It is currently empty! Allow some time for the internal synthesizer to produce more.</span>"
 
+/obj/item/reagent_containers/borghypo/AltClick(mob/living/user)
```
Comment (nemvar) [inline code/modules/reagents/reagent_containers/borghydro.dm]: Call parent.

**Comment by ExcessiveUseOfCobblestone** [comment]
It's just suppose to be a diluent. For patches this is just immersion fluff, for syringes/gels its an actual diluent.

Nutrient/Vitamin healing sounds like an issue I need to fix, but not this PR.

**Comment by shellspeed1** [comment]
> 
> 
> It's just suppose to be a diluent. For patches this is just immersion fluff, for syringes/gels its an actual diluent.
> 
> Nutrient/Vitamin healing sounds like an issue I need to fix, but not this PR. That said, nothing stops you from taking all 3 simultaneously.

Consider that nutriment and vitamin can make you fat which can cause mood debuffs and massive slowdown if you take more than a few units. I don't think they are as strong as you think consider the side effects of using it for anything beyond healing some minor injuries.

**Comment by wesoda25** [comment]
Idk why everyone is complaining, this is precisely the chem we wanted. However, Cobby you should bump up the heal rates to at least 0.25 for granitbitluri, anything below that is absurd. Also work on your names.

**Comment by wesoda25** [comment]
Also you should flesh out more chems like gran, turn them into base level healers... a first tier of sorts....

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm going to change the name of multiple healing to be a generic one rather than a conglom, but in a sep PR since I don't want name discussion clouding balance

**Comment by optimumtact** [comment]
To be honest literally every source of healing in the game needs to be reviewed and most of them probably need to be culled

**Comment by KetasDotepasto** [comment]
Please also remove the white medkits. The bruise packs and burn lotions provide instant healing, which makes wound tending surgery superfluous.

**Comment by optimumtact** [comment]
I'd rather have them only heal damage above a 50 or 60% cap so they can still be used to treat small wounds in each dept

**Comment by ExcessiveUseOfCobblestone** [comment]
Yeah once I swing back around to chemland I'll definitely be looking at those (unless someone wants to work on that now).

Right now though i'm shifting gears towards more ways to interact with patients as a doctor not bound by chemistry (looking at you tox/ox/organs)

**Comment by adamkad1** [comment]
just port baymed, or atleast port peridaxon, or other organ healing chem...

**Comment by ExcessiveUseOfCobblestone** [comment]
no

**Pull request merged**